admin/query/src/main/java/org/codice/ditto/replication/admin/query/sites/fields/IsRemoteManagedField.java

```diff
@@ -15,14 +15,14 @@
 
 import org.codice.ddf.admin.common.fields.base.scalar.BooleanField;
 
-public class IsDisabledLocalField extends BooleanField {
+public class IsRemoteManagedField extends BooleanField {
```

Comment:
✏️ Should we just name the class `RemoteManageField`? `is` is a prefix for a getter method.
❗️ javadoc 😄